arcjet/index.ts

```diff
@@ -521,13 +522,24 @@ type BotOptionsDeny = {
 
 export type BotOptions = BotOptionsAllow | BotOptionsDeny;
 
-export type EmailOptions = {
+type EmailOptionsAllow = {
```

Comment:
```suggestion
export type EmailOptionsAllow = {
```
Need to export this to make documentation easier

Reply:
Fixed this, is there any reason this isn't the case for `BotOptionsAllow` and `BotOptionsDeny`?

Reply:
Those need to be exported (among others). I discovered this while working on the `<DisplayType />` component in our docs. Do you want to open an issue?